src/jail_rules.c

```diff
@@ -1070,6 +1166,24 @@ jail_rules_restrict(
     return NULL;
 }
 
+const char*
+jail_rules_get_value(
```

Comment:
I suspect that this is unnecessary redirection so it would be fine without this in the plugin interface (headers in include directory). I mean anything priv is internal to sailjaild and can be used directly within sailjail code.

Reply:
> This works. I had one comment about jail_rules_get_value() as I think it's not really needed here. Also it might be useful to prevent use of dots at the beginning or end of organization name.

The dots in the beginning are checked, [they must start with a letter](https://github.com/sailfishos/sailjail/pull/42/files#diff-076b99754c3d2cbe30939b2aec4168babc3b93a76e319dd284b714a6edfcaabcR487) but that end can be checked as well I agree.

Reply:
> This works. I had one comment about jail_rules_get_value() as I think it's not really needed here. Also it might be useful to prevent use of dots at the beginning or end of organization name.

The dots in the beginning are checked, [they must start with a letter](https://github.com/sailfishos/sailjail/pull/42/files#diff-076b99754c3d2cbe30939b2aec4168babc3b93a76e319dd284b714a6edfcaabcR487) but that end can be checked as well I agree.

Reply:
I thought this is correct place with its header to have something in the rules struct to be retrieved from. Would it really be a bad thing if this is available for plugins as well, to get the app and org names?

It is anyways used only in jail_run.c.

Reply:
No, not a bad thing per se. Just felt redundant, especially if we are removing the plugin interface in future as sailjaild replaces the functionality it provided.